mkdocs/docs/getting-started/quick-start.md

```diff
@@ -1214,3 +1638,10 @@ When the experiment completes, you will see the experiment stage change from `Ru
     kubectl delete -f $ITER8/samples/knative/quickstart/experiment.yaml
     kubectl delete -f $ITER8/samples/knative/quickstart/experimentalservice.yaml
     ```
+
+=== "Seldon"
```

Comment:
Please include `fortio.yaml` also in the cleanup.

Reply:
yes